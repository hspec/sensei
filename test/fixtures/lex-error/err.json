{"version":"1.0","ghcVersion":"ghc-9.10.0","span":{"file":"test/fixtures/lex-error/Foo.hs","start":{"line":3,"column":11},"end":{"line":3,"column":11}},"severity":"Error","code":21231,"message":["lexical error in string/character literal at character '\\n'"],"hints":[]}
