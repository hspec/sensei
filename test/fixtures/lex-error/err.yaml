version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/lex-error/Foo.hs
  start:
    line: 3
    column: 7
  end:
    line: 3
    column: 11
severity: Error
code: 21231
message:
- lexical error at character '\n'
hints: []
