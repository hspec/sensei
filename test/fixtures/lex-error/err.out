test/fixtures/lex-error/Foo.hs:3:11: error: [GHC-21231]
    lexical error in string/character literal at character '\n'

